src/java/Freight_transport.java

```diff
@@ -0,0 +1,30 @@
+package java;
+
+import java.util.Scanner;
+
+public class Freight_transport extends Land_transport{
+    double CarryingCapacity;
+
+
+    @Override
+    void Vyvod() {
+        super.Vyvod();
+        System.out.println("Carrying capacity: " + CarryingCapacity);
+    }
+    void loading(){
+        System.out.println("Enter the weight of the cargo: ");
+        Scanner scanner=new Scanner(System.in);
```

Comment:
@Hlieb пробелы и операторы
```suggestion
        Scanner scanner = new Scanner(System.in);
```